utils/chemiscope_input.py

```diff
@@ -210,3 +210,54 @@ def write_chemiscope_input(filename, frames, meta=None, extra=None, cutoff=None)
     else:
         with open(filename, 'w') as file:
             json.dump(data, file)
+
+helpstring = """
+    Command-line utility to generate an input for chemiscope  - an interactive 
+    structure-property explorer. Parses an input file containing atomic structures
+    using the ASE I/O module, and converts it into a JSON file that can be loaded in
+    chemiscope. Frame and environment properties must be written in the same file 
+    containing atomic structures: we recommend the extended xyz format, which is 
+    flexible and simple. In all cases, this utility will simply transfer to the
+    JSON input anything that is readable by ASE. 
+"""
+            
+if __name__ == '__main__':
+
+    # command-line execution. requires ASE IO module
+    import argparse
+    try:
+        import ase.io as ase_io
+    except:
+        raise ImportError("chemiscope_input needs ASE modules to parse structure inputs")
+    
+    parser = argparse.ArgumentParser(description=helpstring)
+    parser.add_argument('-i', '--input', type=str, required=True,
```

Comment:
I would use a positional argument for the input, so that the simplest way of using this is `chemiscope_input.py file.xyz` instead of `chemiscope_input.py -i file.xyz`